### PR TITLE
[stable/nfs-server-provisioner] Some fixes

### PR DESCRIPTION
**Pull request by kvaps**
#### Is this a new chart
no

#### What this PR does / why we need it:

* Includes https://github.com/helm/charts/pull/17547 for updates `apiVersion` for `stable/nfs-server-provisioner` that's makes possible to deploy it on Kubernetes 1.16
* Fixes `imagePullSecrets` which are not working for `stable/nfs-server-provisioner`
* Changes default value for rpcbind port from **51413** to **111** for `stable/nfs-server-provisioner`, that's makes possible to discover shares over NFSv3.
* Fix indents, use `nindent` instead
* add @kvaps to OWNERS of nfs-server-provisioner

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by liamdawson** [comment]
I've been eagerly awaiting https://github.com/helm/charts/pull/17547, and looks like this contains the same fix. Any idea if either will be ready Soon? 🙂 

Or is someone likely to maintain it somewhere in the post-stable world I'm hearing about?

**Comment by verwilst** [comment]
Since this touches both nfs-client and nfs-server, and i only use/maintain nfs-client, i'm not comfortable with changing default ports for the server, since i don't know what might break. If you split off your nfs-client changes, i can go ahead and merge those though.

**Comment by kvaps** [comment]
OK, PR rebased, now it includes fixes only for nfs-server-provisioner and does not conflicts with #17547

Changes for nfs-client-provisioner was moved to new PR https://github.com/helm/charts/pull/18442

File: stable/nfs-server-provisioner/values.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ service:
 
   nfsPort: 2049
   mountdPort: 20048
-  rpcbindPort: 51413
+  rpcbindPort: 111
```
Comment (davidkarlsen) [inline stable/nfs-server-provisioner/values.yaml:24]: Don't you need root/privileged access in order to open low-ports (<1024?)

Comment (kvaps) [reply stable/nfs-server-provisioner/values.yaml:24]: Hi, the nfs-server-provisioner was [not designed to run as non root](https://github.com/nfs-ganesha/nfs-ganesha/issues/426#issuecomment-496526616). It also requires some additional capabilities to make possible use VFS:

https://github.com/helm/charts/blob/bc0f5d09fdfcbd1d18575c0dc6da72f5893f8794/stable/nfs-server-provisioner/templates/statefulset.yaml#L50-L54

However you don't need any extra capabilities to bind on 111 port as root user.

Any way the upstream project uses `111` port for rpcbind:
https://github.com/kubernetes-incubator/external-storage/blob/2ba1b775cefa4b7393d805f7b98812c62eb1c167/nfs/deploy/kubernetes/statefulset.yaml#L34-L35

File: stable/nfs-server-provisioner/templates/statefulset.yaml [hunk 2]
```diff
@@ -26,6 +26,10 @@ spec:
       # NOTE: This is 10 seconds longer than the default nfs-provisioner --grace-period value of 90sec
       terminationGracePeriodSeconds: 100
       serviceAccountName: {{ if .Values.rbac.create }}{{ template "nfs-provisioner.fullname" . }}{{ else }}{{ .Values.rbac.serviceAccountName | quote }}{{ end }}
+      {{- if .Values.imagePullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.imagePullSecrets | indent 8 }}
```
Comment (naseemkullah) [inline stable/nfs-server-provisioner/templates/statefulset.yaml]:
```suggestion
        {{- toYaml .Values.imagePullSecrets | nindent 8 }}
```

**Comment by naseemkullah** [comment]
Hi @kvaps please fix conflicts and we'll get this in! :) Thanks.

**Comment by naseemkullah** [comment]
Also @kvaps if you'd be interested in adding yourself as an owner to this chart please feel free to.

File: stable/nfs-server-provisioner/templates/statefulset.yaml (hunk 2 above)
Comment (kvaps) [reply stable/nfs-server-provisioner/templates/statefulset.yaml]: solved in https://github.com/helm/charts/pull/18253/commits/552cd97840e943534c83dbdb0c717d08d8915de9

**Comment by kvaps** [comment]
Closing due the changes are moved in https://github.com/helm/charts/pull/19260

**Pull request closed without merging**